### PR TITLE
ARM64 support for Universal

### PR DESCRIPTION
**Pull request by JunielKatarn**
Enable building for the `ARM64` platform (Universal projects only).

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/3909)

### CODE REVIEW

File: .ado/windows-vs-pr.yml [hunk 1]
```diff
@@ -51,6 +51,10 @@ jobs:
           BuildConfiguration: Release
           BuildPlatform: arm
           UseRNFork: true
+        Arm64Release:
+          BuildConfiguration: Release
+          BuildPlatform: arm64
+          UseRNFork: true
```
Comment (kmelmon) [inline .ado/windows-vs-pr.yml]: Consider disabling the ArmRelease flavor, I don't know of any customer for this and we'll save precious disk space on the CI machines.

**Comment by kmelmon** [comment]
Can you also add ARM64 support to the CLI template project (for apps generated by react-native windows)?  See:
vnext\local-cli\generator-windows\templates\cs\proj
and
vnext\local-cli\generator-windows\templates\cpp\proj

To test see:  [https://github.com/microsoft/react-native-windows/wiki/Contributing-to-the-CLI](https://github.com/microsoft/react-native-windows/wiki/Contributing-to-the-CLI)

**Comment by JunielKatarn** [comment]
> Can you also add ARM64 support to the CLI template project (for apps generated by react-native windows)? See:
> vnext\local-cli\generator-windows\templates\cs\proj
> and
> vnext\local-cli\generator-windows\templates\cpp\proj
> 
> To test see: https://github.com/microsoft/react-native-windows/wiki/Contributing-to-the-CLI

Done.

**Comment by kmelmon** [comment]
Thanks!

---
In reply to: [576866631](https://github.com/microsoft/react-native-windows/pull/3909#issuecomment-576866631) [](ancestors = 576866631)

**Comment by JunielKatarn** [comment]
Blocked by https://github.com/ReactWindows/yoga/pull/1

File: vnext/Common/Common.vcxproj [hunk 2]
```diff
@@ -25,6 +25,14 @@
       <Configuration>Release</Configuration>
       <Platform>ARM</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|ARM64">
```
Comment (NickGerleman) [inline vnext/Common/Common.vcxproj]: @JunielKatarn Would you be willing to pull these out into a separate prop file? I'm about to add a new project, and this duplication is getting to be a bit much.

Comment (JunielKatarn) [reply vnext/Common/Common.vcxproj]: Sorry. I had this in `automerge` and just got in.

**Pull request merged**